components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/utils/SCIMConfigProcessorTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.scim2.common.utils;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.charon3.core.exceptions.CharonException;
+
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+public class SCIMConfigProcessorTest {
+    private SCIMConfigProcessor scimConfigProcessor;
+    private AuthenticationSchema authenticationSchema;
+
+    @DataProvider(name = "facebook-authenticator-all") public static Object[][] propertyProvider() {
```

Comment:
Change testng annotations above the method as follows. Becoz ins nice and clear to understand
@DataProvider(name = "facebook-authenticator-all") 
  public static Object[][] propertyProvider() {
      return new Object[][] { { "Name", "SCIM2" }, { "Address", null } };
  }

Reply:
I wrote like that. bt when I format the code using wso2 code style, it becomes like my code.